cpp/modmesh/view/wrap_view.cpp

```diff
@@ -109,25 +109,34 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapRLine
 
 }; /* end class WrapRLine */
 
-class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapRPythonText
-    : public WrapBase<WrapRPythonText, RPythonText>
+class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapRPythonConsoleDockWidget
+    : public WrapBase<WrapRPythonConsoleDockWidget, RPythonConsoleDockWidget>
 {
 
     friend root_base_type;
 
-    WrapRPythonText(pybind11::module & mod, char const * pyname, char const * pydoc)
+    WrapRPythonConsoleDockWidget(pybind11::module & mod, char const * pyname, char const * pydoc)
         : root_base_type(mod, pyname, pydoc)
     {
 
         namespace py = pybind11;
 
         (*this)
-            .def_property("code", &wrapped_type::code, &wrapped_type::setCode)
+            .def_property(
+                "command",
+                [](wrapped_type const & self)
+                {
+                    return self.command().toStdString();
```

Comment:
Later when we have sufficiently many such wrappers, we should add a caster for `QString`.

Reply:
maybe open an issue, otherwise it's easy to forget lol

Reply:
Sounds like a good idea.  After filing one could you tag it here?